module/Accounting/src/Accounting/Controller/IndexController.php

```diff
@@ -8,6 +8,7 @@ class IndexController extends AbstractActionController
 {
 	public function indexAction()
 	{
+		$this->layout()->setVariable('orgId', $this->params('orgId'));
```

Comment:
Ho usato il plugin Zend\Mvc\Controller\Plugin\Layout per recuperare l'oggetto ViewModel attualmente in uso; se non è presente, viene creato un nuovo oggetto ViewModel.